src/main/java/seedu/duke/Duke.java

```diff
@@ -0,0 +1,47 @@
+package seedu.duke;
+
+public class Duke {
+    private final Storage storage;
+    private final Ui ui;
+    private final Parser parser;
+    private TaskList tasks;
+
+    private static final String[] LIST_OF_COMMANDS =
+            new String[]{"list", "bye", "todo", "mark", "unmark", "event", "deadline", "delete", "find"};
+
+    private static final String SAVE_FILE_NAME = "duke.txt";
+
+    enum Commands {
+        list, bye, todo, mark, unmark, event, deadline, delete, find
+    }
```

Comment:
Will this be better if you use a separate Enumeration class instead of putting it inside your main Duke class?

Reply:
Hmm yeah that's true although I'm not sure if my Enum class will have any methods though. Thanks for the advice!